mhs/outbound/outbound/request/synchronous/tests/test_handler.py

```diff
@@ -148,6 +153,7 @@ def test_post_message_with_correlation_id_passed_in(self, mock_message_id, mock_
 
         mock_message_id.set.assert_called_with(MOCK_UUID)
         mock_correlation_id.set.assert_called_with(correlation_id)
+        mock_interaction_id.set.assert_called_with(INTERACTION_NAME)
```

Comment:
Optional: I mean, technically you are testing the interaction ID contextvar gets set, but it's not really obvious from the test name.

Reply:
I've split this test into two since the first one was asserting quite a lot, and as a result the new one has a more descriptive name